Source/RapyutaSimulationPlugins/Public/Sensors/RRROS2CameraComponent.h

```diff
@@ -115,9 +116,14 @@ class RAPYUTASIMULATIONPLUGINS_API URRROS2CameraComponent : public URRROS2BaseSe
     UPROPERTY(EditAnywhere, BlueprintReadWrite)
     EROS2CameraType CameraType = EROS2CameraType::RGB;
 
+    // Allow to disable rendering (for performance, it still publishes still image)
     UPROPERTY(EditAnywhere, BlueprintReadWrite)
     bool Render = true;
 
+    // Allow to disable publishing (no image will be published)
+    UPROPERTY(EditAnywhere, BlueprintReadWrite)
+    bool Publish = true;

```

Comment:
it maybe better to just set Frequency=0?

Reply:
Oh, yeah, didn't know that you can do that. And besides, that was not supposed to be there. We are using that in a blueprint to stop publishing, but it was supposed to be just in our patch. Was not meant to be in this PR. I will fix that this weekend together with the other inconsistencies with formatter that you mentioned.

Reply:
Got it thank you.
You can access publisher from sensor component and can call
https://github.com/rapyuta-robotics/rclUE/blob/88ec47223c4d3aeb2be34415ec3d067f0c7a6982/Source/rclUE/Public/ROS2Publisher.h#L147

you can add stoppublish fucntion which call above method from basesensorcomponent